pages/common/packer.md

```diff
@@ -10,3 +10,7 @@
 - Check the syntax of a Packer image config:
 
 `packer validate {{path/to/config.json}}`
+
+- Format a Packer image config:
+
+`packer fmt {{path/to/config.json}}`
```

Comment:
```suggestion
`packer fmt {{path/to/config.pkr.hcl}}`
```

The docs say that this command has no effect if a json file is passed. Thats why I suggest changing the file extension in the example. See: https://developer.hashicorp.com/packer/docs/commands/fmt